test/format/util.c

```diff
@@ -505,8 +506,12 @@ alter(void *arg)
 		    "access_pattern_hint=%s",
 		    access_value ? "random" : "none"));
 		access_value = !access_value;
-		if (session->alter(session, g.uri, buf) != 0)
-			break;
+		/*
+		 * Alter can return EBUSY if concurrent with other operations.
+		 */
+		while ((ret = session->alter(session, g.uri, buf)) != 0 &&
+		    ret != EBUSY)
+			testutil_die(ret, "session.alter");
```

Comment:
Maybe the `__wt_yield` calls for compact, backup & alter should be sleep? With yield (and we don't even have a yield in the alter call), we're just pounding on the system.